build/Common.nonprod.props

```diff
@@ -25,6 +25,10 @@
     <ProduceReferenceAssemblyInOutDir>true</ProduceReferenceAssemblyInOutDir>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MicrosoftCodeCoveragePkgVer>[17.4.1]</MicrosoftCodeCoveragePkgVer>
```

Comment:
@utpilla I think this can be further simplified:
1. we can remove these 3 lines.
2. the version is covered by https://github.com/open-telemetry/opentelemetry-dotnet/blob/3274c43bc1031e5b25cfa7a146d428b91abc5032/Directory.Packages.props#L15, we can change the PowerShell script to read from that file.

Reply:
Thanks for the suggestion! Made the changes.